charts/security-profiles-operator/values.yaml

```diff
@@ -0,0 +1,43 @@
+# Default values for security-profiles-operator.
+replicaCount: 3
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  allowPrivilegeEscalation: false
+  readOnlyRootFilesystem: true
```

Comment:
I think changing those may break the operator, right? So do we wanna make it not configurable?